qualtran/_infra/data_types.py

```diff
@@ -184,28 +172,46 @@ def num_qubits(self):
 
 
 @attrs.frozen
-class QFixedPoint(QDType):
-    """Fixed point type to represent real numbers.
+class QFxp(QDType):
+    r"""Fixed point type to represent real numbers.
 
-    The integer part of the float and the fractional part (the part after the
-    decimal plase) can be chosen to have different bitsizes. An additional sign bit is assumed.
+    A real number can be approximately represented in fixed point using num_int
```

Comment:
nit: Escape variable names and equations in the docstring. Eg: `num_int` instead of num_int etc.

Reply:
Done.